### PR TITLE
Feature/trlst 262 bump runtime

### PR DESCRIPTION
**Pull request by davecharles**
- Bumped Dockerfile python layer to 3.9.2
- Bumped PaaS runtime directive accordingly
- Due to 3.9 api changes bumped gevent to latest
- Bumped gunicorn accordingly
- pip-compile latest generates more readable txt output
  hence significant txt diffs.

### CODE REVIEW

File: trade_remedies_caseworker/trade_remedies_caseworker/settings/base.py [hunk 1]
```diff
@@ -254,98 +257,103 @@
...
-                "class": "logging.StreamHandler",
-                "stream": sys.stdout,
-            },
-            "null": {
-                "class": "logging.NullHandler",
-            },
-            "null": {
-                "class": "logging.NullHandler",
-            },
+
```
Comment (Luisella21) [inline trade_remedies_caseworker/trade_remedies_caseworker/settings/base.py:260]: I thought this was part of a previous PR, changing the logging.

**Review by Luisella21 — APPROVED** [review]
LGTM.

File: trade_remedies_caseworker/trade_remedies_caseworker/settings/base.py (hunk 1 above)
Comment (davecharles) [reply trade_remedies_caseworker/trade_remedies_caseworker/settings/base.py:260]: IIRC we only made the changes to the API service.

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**